test/v200/utils/common/endpoint-test-utils.go

```diff
@@ -68,8 +68,9 @@ func (devfile *TestDevfile) CreateEndpoints() []schema.Endpoint {
 			LogInfoMessage(fmt.Sprintf("   ....... add endpoint %d protocol: %s", i, endpoint.Protocol))
 		}
 
-		endpoint.Secure = GetBinaryDecision()
-		LogInfoMessage(fmt.Sprintf("   ....... add endpoint %d secure: %t", i, endpoint.Secure))
+		value := GetBinaryDecision()
+		endpoint.Secure = &value
+		LogInfoMessage(fmt.Sprintf("   ....... add endpoint %d secure: %t", i, *endpoint.Secure))
```

Comment:
Separate from this PR, but I just noticed the name of this file doesn't match the rest (`endpoint-test-utils.go` vs e.g. `component_test_utils.go`)

Reply:
Thanks, I fixed the file name.